### PR TITLE
FAQ backend

### PR DESCRIPTION
**Pull request by ljukas**

### CODE REVIEW

File: app/controllers/api/v1/faq_controller.rb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    }, :except => [:created_at, :updated_at]
+  end
+
+  def create
+    require_admin_permission AdminPermission::EDITOR
+
+    faq = Faq.new(item_params)
+    if faq.save
+      render :status => 200, :json => {
+          message: 'Successfully created Faq.',
```
Comment (patriksletmo) [inline app/controllers/api/v1/faq_controller.rb]: This message will be displayed to the user, why is it in English?

Comment (patriksletmo) [inline app/controllers/api/v1/faq_controller.rb]: Also see all following instances of English messages.

File: app/models/faq_group.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class FaqGroup < ApplicationRecord
+  has_many :faqs, dependent: :destroy
+
+  def number_of_faqs
```
Comment (patriksletmo) [inline app/models/faq_group.rb]: Is this method used somewhere?

Comment (ljukas) [inline app/models/faq_group.rb]: No, I opted not to use it in the end. Will remove. Was dumb from the start, a .count works just fine.

**Pull request merged**